src/libertem/udf/base.py

```diff
@@ -1014,7 +1034,7 @@ def _init_udfs(self, numpy_udfs, cupy_udfs, partition, roi, corrections, device_
             udf.set_meta(meta)
         return (meta, tiling_scheme, dtype)
 
-    def _run_tile(self, udfs, partition, tile, device_tile):
+    def _run_tile(self, udfs, partition, tile, device_tile, roi):
```

Comment:
I think the `roi` parameter can be removed again, no? It doesn't appear to be used anymore.